### PR TITLE
Added fix to loading a model in wrapper generator

### PR DESCRIPTION
**Pull request by imatiach-msft**
Added fix to loading a model in wrapper generator, added verification (save/load model) in pyspark notebook

### CODE REVIEW

File: notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb [hunk 1]
```diff
@@ -137,8 +137,10 @@
    },
    "outputs": [],
    "source": [
-    "from mmlspark import FindBestModel\n",
-    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)"
+    "from mmlspark import FindBestModel, BestModel\n",
+    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)\n",
+    "bestModel.write().overwrite().save('model.mml')\n",
+    "loadedBestModel = BestModel.load('model.mml')"
```
Comment (drdarshan) [inline notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb]: FindBestModel.load?

Comment (drdarshan) [inline notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb]: Oh I see, ``FindBestModel`` used to return ``FindBestModelModel`` but now returns just ``BestModel``?

File: src/codegen/src/main/scala/PySparkWrapper.scala [hunk 2]
```diff
@@ -149,6 +149,7 @@ abstract class PySparkWrapper(entryPoint: PipelineStage,
         |    @staticmethod
         |    def _from_java(java_stage):
         |        stage_name=${entryPointName}.__module__
```
Comment (drdarshan) [inline src/codegen/src/main/scala/PySparkWrapper.scala]: Please clarify that this is in fact ``module_name``

File: src/codegen/src/main/scala/PySparkWrapper.scala [hunk 3]
```diff
@@ -149,6 +149,7 @@ abstract class PySparkWrapper(entryPoint: PipelineStage,
...
+        |        stage_name = stage_name.rsplit(".", 1)[0] + ".${entryPointName}"
```
Comment (drdarshan) [inline src/codegen/src/main/scala/PySparkWrapper.scala]: module_name.rsplit(".", 1)[0] + ...

File: notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb [hunk 4]
```diff
@@ -137,8 +137,10 @@
    },
    "outputs": [],
    "source": [
-    "from mmlspark import FindBestModel\n",
-    "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=lrmodels).fit(test)"
+    "from mmlspark import FindBestModel, BestModel\n",
```
Comment (drdarshan) [inline notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb]: Please add similar tests to some of the other notebooks to validate that models produced by other estimators can be loaded as well.

File: src/codegen/src/main/scala/PySparkWrapper.scala (hunk 3 above)
Comment (drdarshan) [inline src/codegen/src/main/scala/PySparkWrapper.scala]: and please use " = " or "=" consistently..

File: notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb (hunk 1 above)
Comment (imatiach-msft) [inline notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb]: yes, that's correct

File: src/codegen/src/main/scala/PySparkWrapper.scala (hunk 2 above)
Comment (imatiach-msft) [inline src/codegen/src/main/scala/PySparkWrapper.scala]: done

File: src/codegen/src/main/scala/PySparkWrapper.scala (hunk 3 above)
Comment (imatiach-msft) [inline src/codegen/src/main/scala/PySparkWrapper.scala]: done

File: notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb (hunk 4 above)
Comment (imatiach-msft) [inline notebooks/samples/201 - Amazon Book Reviews - TextFeaturizer.ipynb]: done, added more tests

**Comment by imatiach-msft** [comment]
I fixed the quoting to be consistent

**Pull request merged**